### PR TITLE
Document new VEP option --individual_zyg (e111)

### PR DESCRIPTION
**Pull request by dglemos**
New option available for release 111
[ENSVAR-4523](https://www.ebi.ac.uk/panda/jira/browse/ENSVAR-4523)

Sandbox: http://wp-np2-11.ebi.ac.uk:7040/info/docs/tools/vep/script/vep_options.html#output

### CODE REVIEW

File: docs/htdocs/info/docs/tools/vep/script/vep_options.html [hunk 1]
```diff
@@ -892,6 +892,21 @@ <h2 id="output">Output options</h2>
...
+        Consider alternate and reference alleles present in the genotypes of the
+        specified individual(s). May be a single individual, a
+        comma-separated list or "all" to assess all individuals
+        separately. Returns a list of individuals and their zygosity.
+        Only works with VCF files containing individual genotype data;
+        individual IDs are taken from column headers. <i>Not used by
+        default</i>
+      </td>
+      <td>ZYG</td>
+      <td>&nbsp;</td>
```
Comment (nuno-agostinho) [inline docs/htdocs/info/docs/tools/vep/script/vep_options.html:908]: Mention incompatibility with `--individual`:

```suggestion
      <td>
        <div style="padding:2px 0px"><a href="#opt_individual">--individual</a></div>
      </td>
```

File: docs/htdocs/info/docs/tools/vep/script/vep_options.html [hunk 2]
```diff
@@ -892,6 +892,21 @@ <h2 id="output">Output options</h2>
         <div style="padding:2px 0px"><a href="#opt_minimal">--minimal</a></div>
```
Comment (nuno-agostinho) [inline docs/htdocs/info/docs/tools/vep/script/vep_options.html:892]: Mention incompatibility with `--individual_zyg`

```suggestion
        <div style="padding:2px 0px"><a href="#opt_minimal">--minimal</a></div>
        <div style="padding:2px 0px"><a href="#opt_individual_zyg">--individual_zyg</a></div>
```

**Review by nuno-agostinho — CHANGES_REQUESTED** [review]
Hey @dglemos, thanks for the update! I think it is only missing information on incompatible options, but seems fine otherwise.

**Review by nuno-agostinho — APPROVED** [review]
Thank you for the changes, @dglemos!

File: docs/htdocs/info/docs/tools/vep/script/vep_options.html [hunk 3]
```diff
@@ -890,6 +890,24 @@ <h2 id="output">Output options</h2>
       <td>IND, ZYG</td>
       <td>
         <div style="padding:2px 0px"><a href="#opt_minimal">--minimal</a></div>
+        <div style="padding:2px 0px"><a href="#opt_individual_zyg">--individual_zyg</a>
```
Comment (jgtate) [inline docs/htdocs/info/docs/tools/vep/script/vep_options.html:893]:
```suggestion
        <div style="padding:2px 0px"><a href="#opt_individual_zyg">--individual_zyg</a></div>
```

**Pull request merged**